code/stable/dblpend/TraceyGraph/allvsr.dot

```diff
@@ -63,16 +63,16 @@ digraph allvsr {
 	{theory:angleIM1, theory:angleIM2}
 	}
 
+	instance:correct	[shape=box, color=black, style=filled, fillcolor=ivory, label="NFR:correct"];
+	instance:portable	[shape=box, color=black, style=filled, fillcolor=ivory, label="NFR:portable"];
 	instance:inputValues	[shape=box, color=black, style=filled, fillcolor=ivory, label="FR:inputValues"];
 	instance:verifyInptVals	[shape=box, color=black, style=filled, fillcolor=ivory, label="FR:verifyInptVals"];
 	instance:calcAng	[shape=box, color=black, style=filled, fillcolor=ivory, label="FR:calcAng"];
 	instance:outputValues	[shape=box, color=black, style=filled, fillcolor=ivory, label="FR:outputValues"];
-	instance:correct	[shape=box, color=black, style=filled, fillcolor=ivory, label="NFR:correct"];
-	instance:portable	[shape=box, color=black, style=filled, fillcolor=ivory, label="NFR:portable"];
 
-	subgraph FR {
+	subgraph NFR {
 	rank="same"
-	{instance:inputValues, instance:verifyInptVals, instance:calcAng, instance:outputValues, instance:correct, instance:portable}
+	{instance:correct, instance:portable, instance:inputValues, instance:verifyInptVals, instance:calcAng, instance:outputValues}
 	}
```

Comment:
Rather than having two subgraphs; FRs and NFRs. It looks like the previous `.dot` diagrams were broken, intermixing the two and using a label that only mentions one.

Reply:
(I see this as a "win" of this PR as we're uncovering some logical issues rather than something we need to scramble to "fix" in this PR.)

Reply:
Good!